infra/modules/vercel-application/vercel.tf

```diff
@@ -10,6 +10,9 @@ resource "vercel_project" "this" {
 
   build_command  = var.build_command
   root_directory = var.root_directory
+
+  # Do not run on renovate pull requests
+  ignore_command = "[ \"$VERCEL_GIT_COMMIT_AUTHOR_LOGIN\" == \"renovate[bot]\" ]"
```

Comment:
ignore_command = "[ \"$VERCEL_GIT_COMMIT_AUTHOR_LOGIN\" == \"renovate[bot]\" && turbo-ignore ]"

Reply:
Perhaps npx turbo-ignore?

Reply:
Yeah probably. Thought you didn't need the npx